### PR TITLE
Feature to invoke core bitops utilities from plugins

### PR DESCRIPTION
**Pull request by shyamrayaprolu**
# Description

Added new feature in `plugins.py` to invoke core `bitops` utilities from plugins

Fixes # (issue)

`plugins.py`

## Type of change

- [X] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: scripts/plugins.py [hunk 1]
```diff
@@ -45,6 +48,8 @@
         Deploy_Plugins()
     elif mode == "install":
         Install_Plugins()
+    elif mode == "schema_parsing":
```
Comment (PhillypHenning) [inline scripts/plugins.py:51]: I think the naming of the command line arg could be improved. 

Off the top of my head; 
- Schema
- Parser
- DeCLI

File: scripts/plugins.py [hunk 2]
```diff
@@ -45,6 +48,8 @@
...
+        Get_Config_List(config_file, schema_file)
```
Comment (PhillypHenning) [inline scripts/plugins.py:52]: Are you doing anything with this list? I believe this function returns the options and cli properties it parses.

File: scripts/plugins.py (hunk 1 above)
Comment (shyamrayaprolu) [reply scripts/plugins.py:51]: reason for picking this is we are only invoking the schema parsing function. So certainly DeCLI and schema would be confusing.

File: scripts/plugins/deploy_plugins.py [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import os
 import subprocess
+from telnetlib import theNULL
```
Comment (PhillypHenning) [inline scripts/plugins/deploy_plugins.py:3]: Is this imported value being used?

**Review by PhillypHenning — APPROVED** [review]
Posted a suggestion for a change, and 2 questions. Otherwise it looks good Shyam

File: scripts/plugins.py (hunk 2 above)
Comment (shyamrayaprolu) [reply scripts/plugins.py:52]: Not sure what this function does entirely. I was under the impression, it just exports all the configs into OS ENV variables. Correct me if this is NOT what this function does.

Comment (shyamrayaprolu) [reply scripts/plugins.py:52]: I am trying to find a simple way to get this utility function to be used from the plugin. Also this is for one time use only as we will eventually move away from performing this functionality from plugins. If this is complex then I would not make any changes to plugin.py and use the helm plugin as is (which does parsing today)

File: scripts/plugins/deploy_plugins.py (hunk 3 above)
Comment (shyamrayaprolu) [reply scripts/plugins/deploy_plugins.py:3]: Not sure what this does.

**Pull request merged**